<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="model">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>1241</width>
      <height>961</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="scale">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>500</y>
      <width>291</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Scale</string>
    </property>
    <widget class="QLabel" name="scaleZ">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Z:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="scaleLineX">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="scaleLineY">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="scaleX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="scaleLineZ">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="scaleY">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnScale">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="move">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>500</y>
      <width>291</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Move</string>
    </property>
    <widget class="QLabel" name="moveZ">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Z:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="moveLineX">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="moveLineY">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="moveX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="moveLineZ">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="moveY">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMove">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Move</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Rotate">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>700</y>
      <width>301</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Rotate</string>
    </property>
    <widget class="QLabel" name="rotateZ">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Z:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="rotateLineX">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="rotateLineY">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="rotateX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="rotateLineZ">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="rotateY">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRotateY">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>80</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotate Y</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRotateX">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>80</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotate X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRotateZ">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>120</y>
       <width>80</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotate Z</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="fileName">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>491</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(208, 188, 130);</string>
    </property>
    <property name="text">
     <string>Path</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpenFile">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>350</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Normalization">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>301</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Normalization</string>
    </property>
    <widget class="QLabel" name="normalZ">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Step:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="normalLineMax">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="normalLineMin">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="normalX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Max:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="normalLineStep">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>130</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="normalY">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Min:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnNormal">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>80</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="errorLabel">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>390</y>
      <width>641</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>error text</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
